automatic-off-session-payment/process.edn

```diff
@@ -0,0 +1,169 @@
+{:format :v3,
+ :transitions
+ [{:name :transition/enquire,
```

Comment:
In other processes `enquire` transition just creates a new transaction, with the idea that then there can be messages exchanged upon that transaction but no actions are attached to it. Should we call this transition something else in this process in order to avoid confusion? `request-payment` is usually the transition that requests a booking/stock reservation but here no payment is yet requested. How about `request`? What do you think?

Reply:
Good point! Updated to `:transition/request-booking`